jbosstools/configuration/ide-config.properties

```diff
@@ -94,6 +82,65 @@ jboss.fuse.extras.site.url|devstudio|12=https://devstudio.redhat.com/12/stable/u
 
 ##Discovery
 # =========
+# 4.14.0.Final / eclipse 4.14 (2019-12)
+jboss.discovery.directory.url|jbosstools|4.14=http://download.jboss.org/jbosstools/photon/stable/updates/discovery.earlyaccess/4.14.0.Final/jbosstools-directory.xml
+#jboss.discovery.site.url|jbosstools|4.14=http://download.jboss.org/jbosstools/photon/stable/updates/
```

Comment:
Why is this commented ? What will happen on those versions ?

Reply:
fixed.